tests/acl/test_acl_outer_vlan.py

```diff
@@ -527,6 +572,13 @@ def _do_verification(self, ptfadapter, duthost, tbinfo, vlan_setup_info, ip_vers
                                     pkt_type=pkt_type,
                                     tagged_mode=tagged_mode,
                                     stage=stage)
+        if stage == INGRESS:
+            # Wait mac
+            pytest_assert(wait_until(30, 1, 0, check_mac_status, duthost,
+                                     ptfadapter, outer_vlan_id, dst_mac, dst_port[0]), "mac table is not updated")
```

Comment:
Is it necessary to build MAC table on DUT for ingress test?

Reply:
yes, one case failed because no mac for PortChannel.